### PR TITLE
Add absolute path checks for keyFile, outputPath and contentPath

### PR DESCRIPTION
**Pull request by sidoruk-sv**
If we determine absolute paths for `keyFile`, `outputPath` and `contentPath` fields, then all will be broken.
This PR add ability to determine paths in absolute manner.

### CODE REVIEW

**Comment by sidoruk-sv** [comment]
@johnagan need any changes to PR?

**Comment by johnagan** [comment]
@sidoruk-sv thanks for the contribution! I'll test this out this weekend.

File: index.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ var fs = require('fs');
 var path = require('path');
 var join = path.join;
 var mkdirp = require('mkdirp');
+var pathIsAbsolute = require('path-is-absolute');
```
Comment (johnagan) [inline index.js]: Do we really need this dependency?

Comment (sidoruk-sv) [inline index.js]: Yeah, this is a debate topic:
- \+ why small npm modules ecosystem is good: https://github.com/sindresorhus/ama/issues/10#issuecomment-117766328
- \-  history with leftpad: http://www.haneycodes.net/npm-left-pad-have-we-forgotten-how-to-program/

Can inline that code inside project, do I need?

Comment (johnagan) [inline index.js]: I guess I was referring to this:
https://nodejs.org/api/path.html#path_path_isabsolute_path

Comment (sidoruk-sv) [inline index.js]: Cool, thanx for a hint)
Rewrited to use native method

**Pull request merged**